autopush/tests/test_web_validation.py

```diff
@@ -158,13 +158,13 @@ def _make_fut(self):
         schema.context["log"] = Mock()
         return schema
 
-    def _make_test_data(self, headers=None, body="", path_args=None,
-                        path_kwargs=None, arguments=None):
+    def _make_test_data(self, headers=None, body="", args=None, kwargs=None,
+                        arguments=None):
```

Comment:
I had the same thought and forgot all about it, I was thinking of renaming arguments (how about request_arguments?) it's rarely used anyway